src/prefect/environments/storage/docker.py

```diff
@@ -79,19 +72,50 @@ def __init__(
         self.image_name = image_name
         self.image_tag = image_tag
         self.python_dependencies = python_dependencies or []
+        self.python_dependencies.append("wheel")
+
         self.env_vars = env_vars or {}
+        self.env_vars["PREFECT__USER_CONFIG_PATH"] = "/root/.prefect/config.toml"
+
         self.files = files or {}
         self.flows = dict()  # type: Dict[str, str]
         self._flows = dict()  # type: Dict[str, "prefect.core.flow.Flow"]
         self.base_url = base_url or default_url
         self.local_image = local_image
+        self.extra_commands = []
 
         version = prefect.__version__.split("+")
         if prefect_version is None:
             self.prefect_version = "master" if len(version) > 1 else version[0]
         else:
             self.prefect_version = prefect_version
 
+        if base_image is None:
+            python_version = "{}.{}".format(
+                sys.version_info.major, sys.version_info.minor
+            )
+            if re.match("^[0-9]+\.[0-9]+\.[0-9]+$", self.prefect_version) != None:
+                # note: this does not necessarily mean that we have built/pushed all previous prefect versions to dockerhub
```

Comment:
luckily a user will only hit this block of code if they're running 0.6.8+ so we don't have to retroactively build old images with this pattern

Reply:
true, but this won't stop them from specifying an older version [here](https://github.com/PrefectHQ/prefect/blob/7576d7ba0b9c813d9af2004a67c74ef7a458a60d/src/prefect/environments/storage/docker.py#L62) (such as 0.6.6) when using 0.6.8+ locally. I don't see why someone would want to do this, but they could I suppose. 

Though, I take your larger point... I'm willing to let the docker pull fail like we would do for any other user-specified `base_image`. I'll take out the code comment.